src/components/CategoryList.jsx

```diff
@@ -11,7 +11,10 @@ const CategoryList = ({ className }) =>
         variant="category"
         label={category.label}
         icon={category.iconName}
-        iconStyle={{ backgroundColor: category.backgroundColor }}
+        iconStyle={{
+          color: `var(--${category.color})`,
+          backgroundColor: `var(--${category.backgroundColor})`,
```

Comment:
I notice you finally decided to declare both colors instead of generating one as a variant of the other. I'm curious, why? Actually I thought in this PR you would rename `backgroundColor` to `color` and just deduce the new background from it.

Reply:
Yeah this is what I wanted to achieve too. However, I couldn't find a good way to import the (s)css variables in JS with our webpack setup. Do you have any idea ?

Reply:
> Do you have any idea ?

Actually I don't see where these values would be useful in CSS, so I don't get the need for a change. 
IMO the current setup with the values defined in the YAML file is good enough for the quantity of data and the frequency of changes.